src/docfx/lib/JsonUtility.cs

```diff
@@ -125,6 +125,17 @@ public static JObject Merge(JObject first, IEnumerable<JObject> objs)
             return result;
         }
 
+        public static bool IsNullOrEmpty(this JToken token)
+        {
+            return
+                (token == null) ||
+                (token.Type == JTokenType.Array && !token.HasValues) ||
+                (token.Type == JTokenType.Object && !token.HasValues) ||
+                (token.Type == JTokenType.String && string.IsNullOrEmpty(token.ToString())) ||
+                (token.Type == JTokenType.Null) ||
+                (token.Type == JTokenType.Undefined);
```

Comment:
>JTokenType.Undefined [](start = 31, length = 20)

just curious: what does this mean? any example?

Reply:
Unit tests for these have been added to JsonUtilityTest and YamlUtilityTest